modules/iam-primary-roles/tfstate-context.tf

```diff
@@ -18,7 +18,7 @@ variable "tfstate_account_id" {
 
 variable "tfstate_role_arn_template" {
   type        = string
-  default     = "arn:aws:iam::%s:role/%s-%s-%s-%s"
+  default     = "arn:aws-us-gov:iam::%s:role/%s-%s-%s-%s"
```

Comment:
Looks like another spot where you should utilize `data.aws_partition.current.partition`.

Reply:
@Gowiem I did not mean to include these files in the commit